src/components/Calculator.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import '../StyleSheets/calculator.css';
+
+function Calculatorb() {
+  return (
+    <div className="calculator-container">
+      <input type="number" className="number-input" placeholder="0" />
+      <div className="buttons-container">
+        <div className="numbers-container">
+          <button className="calculator-button" type="button">AC</button>
+          <button className="calculator-button" type="button">+/-</button>
+          <button className="calculator-button" type="button">%</button>
+          <button className="calculator-button" type="button">7</button>
+          <button className="calculator-button" type="button">8</button>
+          <button className="calculator-button" type="button">9</button>
+          <button className="calculator-button" type="button">4</button>
+          <button className="calculator-button" type="button">5</button>
+          <button className="calculator-button" type="button">6</button>
+          <button className="calculator-button" type="button">1</button>
+          <button className="calculator-button" type="button">2</button>
+          <button className="calculator-button" type="button">3</button>
+          <button className="calculator-button zero" type="button">0</button>
+          <button className="calculator-button" type="button">.</button>
+        </div>
+        <div className="basic-operations-container">
+          <button className="calculator-button operator" type="button">/</button>
```

Comment:
| 1 | 2 |
|-|-|
| ![image](https://github.com/CarlosZ96/mathmagicians/assets/93039359/44d859dc-5077-463b-b7ec-a515f4b2e2e2) | ![image](https://github.com/CarlosZ96/mathmagicians/assets/93039359/965d5625-8ef5-4be4-97e9-05b2d5c84634)

- [ ] It would be nice if you double-check the given design then you will find two things different from your design 🤔

1. Kindly make sure the color for the white button is the same because I can see they are different in your design and the given design 😨 Kindly make the color of your button the same as the given design 👍
2. It would be great if you put the right symbol for division which you can see in the given design so kindly use the same symbol which is in the microverse design 👍